### DISCUSSION TITLE
Appreciate the effort, but it isn't working on my side

### DISCUSSION THREAD

**Discussion by FartyPants** [vladmandic/sdnext#51] [Q&A]
The install instructions are not good, I could not run this from fresh state (missing things)

**Comment by vladmandic** [comment]
can you elaborate a bit more? what is not working, what is missing?
it could be that you caught it in-between state since i'm moving really fast and quite often break install procedure only to fix it an hour later.

**Comment by bullseyepsa** [comment]
The windows install only worked correctly for me when I ran from PowerShell. I couldn't get it to work from CMD.EXE which I still use like a luddite.

**Comment by vladmandic** [comment]
see #92 - work in progress, tbd soon (this week).

**Comment by vladmandic** [comment]
major update posted, see #99 for details
